modules/tinymce/src/themes/silver/test/ts/module/ToolbarUtils.ts

```diff
@@ -30,7 +30,7 @@ const pAssertFloatingToolbarHeight = async (editor: Editor, expectedHeight: numb
 
 const pOpenFloatingToolbarAndAssertPosition = async (editor: Editor, getTop: () => number, pActions?: () => Promise<void>): Promise<void> => {
   await pOpenMore(ToolbarMode.floating);
-  await pAssertFloatingToolbarPosition(editor, getTop, 105, 531);
+  await pAssertFloatingToolbarPosition(editor, getTop, 105, 520);
```

Comment:
This doesn't seem related to the feature you added 🤔

Reply:
Indeed it seems I needed to make this change to pass some tests locally and forgot to clean it up. I just cleaned it up in a new commit